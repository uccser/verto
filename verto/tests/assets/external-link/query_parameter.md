Check out this [website](https://www.google.com?key=value).
