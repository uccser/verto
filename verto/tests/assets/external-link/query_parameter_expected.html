<p>Check out this <a href="https://www.google.com?key=value" target="_blank">website</a>.</p>
